src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -376,9 +373,11 @@ public static class TagHelperBlockRewriter
                 var htmlSymbol = firstChild.Symbols[firstChild.Symbols.Count - 1] as HtmlSymbol;
                 switch (htmlSymbol.Type)
                 {
-                    // We purposefully do not capture NoQuotes at parse time to protect users from rendering
-                    // dynamic content with spaces that can break attributes.
-                    // Ex: <a my-attribute=@value /> where @value results in the test "hello world".
+                    // Treat NoQuotes and DoubleQuotes equivalently. We purposefully do not persist NoQuotes
+                    // ValueStyles at code generation time to protect users from rendering dynamic content with spaces
+                    // that can break attributes.
+                    // Ex: <tag my-attribute=@value /> where @value results in the test "hello world".
```

Comment:
My earlier comment was that `<tag my-attribute=@value />` example can't be applicable both here and at [line 170](https://github.com/aspnet/Razor/pull/743/commits/e0139a4ed9c5c62d6e4c422d17aa13db8406996f#diff-72fec5f339f6e3ee604fc27d0fc6fd95R170). So what example applies here? Or, if this is for consistency with the earlier code, say that.